pkg/core/native/native_neo.go

```diff
@@ -156,6 +158,20 @@ func (n *NEO) OnPersist(ic *interop.Context) error {
 	if err != nil {
 		return err
 	}
+	prev := n.nextValidators.Load().(keys.PublicKeys)
```

Comment:
Well, this saves us a write, but we probably can also save on `GetValidatorsInternal` call, it only changes if there are transfers or votes.